content/documentation/3.6.x/api/ua.html

```diff
@@ -141,7 +141,9 @@
   "eventHandlers" => "Optional @Object@ of event handlers to be registered to each call event. Define an event handler for each event you want to be notified about.",
   "extraHeaders" => "@Array@ of @Strings@ with extra SIP headers for the INVITE request.",
   "anonymous" => "@Boolean@ field indicating whether the call should be done anonymously. Default value is @false@.",
-  "sessionTimersExpires" => "@Number@ (in seconds) for the default Session Timers interval (default value is 90, do not set a lower value)."
+  "sessionTimersExpires" => "@Number@ (in seconds) for the default Session Timers interval (default value is 90, do not set a lower value).",
+  "fromUserName" => "@String@ to override the user name in the From header.",
+  "fromDisplayName" => "Descriptive name (@String@) to be shown to the called party when calling or sending IM messages. Overrides the UA configuration parameter @display_name@."
```

Comment:
Only applies to calling, you indicate "sending IM messages" which does not apply.

Reply:
Sorry – a stupid copy-paste error!